### PR TITLE
[CLI][Easy] update move run to include more params

### LINKED ISSUE #8827: [CLI] Add more test to "aptos move run" for all different type of argument
We did test most of the string value here - https://github.com/aptos-labs/aptos-core/pull/8823

But there could be alot more, such as u8, address, etc, and nested args

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Add new entry func `test_move_run` for testing purpose. Make sure all other params are abled to run from CLI

resolve #8827 

### Test Plan

```
023-09-05 14:37:50,551 - INFO - These tests passed:
2023-09-05 14:37:50,551 - INFO - test_metrics_accessible
2023-09-05 14:37:50,551 - INFO - test_init
2023-09-05 14:37:50,551 - INFO - test_config_show_profiles
2023-09-05 14:37:50,551 - INFO - test_account_fund_with_faucet
2023-09-05 14:37:50,551 - INFO - test_account_create
2023-09-05 14:37:50,551 - INFO - test_account_lookup_address
2023-09-05 14:37:50,551 - INFO - test_account_resource_account
2023-09-05 14:37:50,551 - INFO - test_aptos_header_included
2023-09-05 14:37:50,551 - INFO - test_move_compile
2023-09-05 14:37:50,551 - INFO - test_move_compile_script
2023-09-05 14:37:50,551 - INFO - test_move_publish
2023-09-05 14:37:50,552 - INFO - test_move_run
2023-09-05 14:37:50,552 - INFO - test_move_view
2023-09-05 14:37:50,552 - INFO - test_stake_initialize_stake_owner
2023-09-05 14:37:50,552 - INFO - test_stake_add_stake
2023-09-05 14:37:50,552 - INFO - test_stake_withdraw_stake_before_unlock
2023-09-05 14:37:50,552 - INFO - test_stake_unlock_stake
2023-09-05 14:37:50,552 - INFO - test_stake_withdraw_stake_after_unlock
2023-09-05 14:37:50,552 - INFO - test_stake_increase_lockup
2023-09-05 14:37:50,552 - INFO - test_stake_set_operator
2023-09-05 14:37:50,552 - INFO - test_stake_set_voter
2023-09-05 14:37:50,552 - INFO - test_stake_create_staking_contract
2023-09-05 14:37:50,552 - INFO - test_node_show_validator_set
2023-09-05 14:37:50,552 - INFO - test_node_update_consensus_key
2023-09-05 14:37:50,552 - INFO - test_node_update_validator_network_address
2023-09-05 14:37:50,552 - INFO - test_account_rotate_key
2023-09-05 14:37:50,552 - INFO - All tests passed!
```

### CODE REVIEW

File: aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move [hunk 1]
```diff
@@ -206,6 +206,20 @@ module addr::cli_e2e_tests {
         create_hero(account, description, gender, name, race, uri);
     }
 
+    public entry fun test_move_run(
```
Comment (xbtmatt) [inline aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move:209]: In the PR description you say `view` func but this is an entry function, just making sure you intended to use an entry function here 😃

Comment (0xjinn) [reply aptos-move/move-examples/cli-e2e-tests/common/sources/cli_e2e_tests.move:209]: oops! I meant entry func. Good catch!

**Review by xbtmatt — APPROVED** [review]
lgtm! Quick question though, what's the easiest way to run these myself? Just to make sure the test succeeds and whatnot

**Comment by banool** [comment]
@xbtmatt check the README in crates/aptos/e2e, it explains it.

**Pull request merged**